setup.cfg

```diff
@@ -0,0 +1,7 @@
+[tox:tox]
+envlist = {py27,py34,py35,py36,py37,pypy,pypy3}
+skipsdist=True
+
+[testenv]
+commands = python setup.py test {posargs}
+setenv = TZ = UTC+5
```

Comment:
I would drop the spaces around the second `=` to get `setenv = TZ=UTC+5`.

(Also it's a bit sad if the tests only pass in one specific timezone, but I don't think that's your fault?)

Reply:
Without this timezone, `zc.FileStorage.tests.test_pack_sleep` does not pass. It is too bad indeed, but in a first time, I just tried to make the tests pass without changing anything.